### PR TITLE
Doc/backstage docs

### PR DESCRIPTION
**Pull request by LinMAD**
Refer documentation folder to the entry readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ traceability and logs in Datadog.
 - [X] gRPC Unary Server
 - [X] HTTP - Echo
 
+## Documentation
+
+There is detailed documentation stored in
+[docs](https://github.com/coopnorge/go-datadog-lib/tree/main/docs).
```
Comment (nhhagen) [inline README.md:21]: Please use a relative link

```suggestion
[docs](docs/).
```

File: .vale/styles/Vocab/General/accept.txt [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+APIs
+API's
+enum
+async
+cfg
+cfg_api
+pdf
+Pdf
+OAuth
+Oauth
```
Comment (nhhagen) [inline .vale/styles/Vocab/General/accept.txt:12]: Stick to a single correct spelling

```suggestion
OAuth
```

File: .vale/styles/Vocab/General/accept.txt [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+github
+Datadog
+DataDog
+datadog
+Mockhandler
+mockhandler
+Eitri
+eitri
+Unary
+unary
```
Comment (nhhagen) [inline .vale/styles/Vocab/General/accept.txt:33]: We should not enable randomish spelling of words, we should stick with the proper ones. Bad spelling reduces the perceived quality of our documentation.

Eg: `github` is spelled `GitHub`, `postgres` is spelled `PostgreSQL`, `oauth` is spelled `OAuth`.

There is not need for duplicate entries to add support for the first letter to be upper case. Instead you write `[Uu]nary`.

There are also words in this list that I could not find in the documentation, these should be removed.

Comment (LinMAD) [reply .vale/styles/Vocab/General/accept.txt:33]: @nhhagen I thought you would like to implement a generic vale spell book so Coop can re-use it, but instead, it feels like we will generate N amounts of dictionaries now?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (LinMAD) [reply README.md:21]: Not sure about that, how does it work? The browser will resolve the link to the docs folder to open nested readme.md? Can you show me documentation about that syntax?

[the remark this answers is not in the document]
Comment (nhhagen) [reply README.md:21]: Sort, the `&` should ve a `/`.

File: .vale/styles/Vocab/General/accept.txt (hunk 3 above)
Comment (nhhagen) [reply .vale/styles/Vocab/General/accept.txt:33]: We have a separate account effort on that, no need to pre-populate.

Ref:
- https://github.com/coopnorge/github-workflow-techdocs/issues/16

**Comment by nhhagen** [comment]
I added a PR with tooling you need to validate the docs locally, faster feedback loop.

- https://github.com/coopnorge/go-datadog-lib/pull/29

**Review by nhhagen — APPROVED** [review]

**Pull request merged**